warehouse/src/main/java/io/skai/warehouse/command/entity/component/InsertOnDuplicateUpdateComponentCommand.java

```diff
@@ -0,0 +1,12 @@
+package io.skai.warehouse.command.entity.component;
+
+import com.kenshoo.pl.entity.InsertOnDuplicateUpdateCommand;
+import io.skai.warehouse.model.component.ComponentEntity;
+
+public class InsertOnDuplicateUpdateComponentCommand extends
+        InsertOnDuplicateUpdateCommand<ComponentEntity, ComponentEntity.Key> {
```

Comment:
inline it please, for future: class declarations always performed as single line